tsl/test/isolation/expected/compression_ddl.out

```diff
@@ -29,22 +29,19 @@ step UnlockChunk: ROLLBACK;
 step I1: <... completed>
 step Ic: COMMIT;
 step C1: <... completed>
-compress_chunk                        
---------------------------------------
-_timescaledb_internal._hyper_1_1_chunk
+compress
+--------
+t       
 (1 row)
 
 step Cc: COMMIT;
-step SC1: SELECT count(*) AS only FROM ONLY _timescaledb_internal._hyper_1_1_chunk; SELECT count(*) FROM _timescaledb_internal._hyper_1_1_chunk;
-only
-----
-   0
-(1 row)
-
-count
------
-   11
-(1 row)
+step SC1: SELECT count_chunktable(ch) FROM show_chunks('ts_device_table') AS ch ORDER BY ch::text;
+count_chunktable
```

Comment:
This output is difficult to interpret. The point of this particular test is to show that compression succeeded and there is no data in the original chunk's table. With the test changes, it is very difficult to interpret what is going on here. I would prefer to see this as 2 output rows/ 2 columns , but limit the output to the chunk that was compressed. This way it is closer to what the original output shows and is easier to figure out what is going on.

Reply:
ok changed